doc/api/cli.md

```diff
@@ -1602,6 +1615,12 @@ and the line lengths of the source file (in the key `lineLengths`).
 }
 ```
 
+### `NO_COLOR=1`
+
+The [`NO_COLOR`][] environment variable is an informal standard supported by
+growing number of applications for disabling ANSI color escape codes. Within
+Node.js, `NO_COLOR` is an alias for `NODE_DISABLE_COLORS`.
```

Comment:
Optional but I think the informal standard stuff is secondary and should be de-emphasized in favor of what-does-this-thing-do:

```suggestion
Alias for `NODE_DISABLE_COLORS`.
```